doc/user_guide.md

```diff
@@ -0,0 +1,117 @@
+# Exasol SQL Statement Builder User Guide
+
+You can follow this guide to learn more about how to use the SQL Statement
+Builder. It provides explanation of currently supported features with various
+code examples.
+
+## Overview
+
+The Exasol SQL Statement Builder abstracts programmatic creation of SQL
+statements and is intended to replace ubiquitous string concatenation solutions
+which make the code hard to read and are prone to errors and security risks.
+
+SQL Statement Builder allows users to create a SQL statement classes using
+[fluent programming][fluent] that can render into a string.
+
+**Currently supported SQL statements:**
+
+- [CREATE TABLE](#create-table)
+- SELECT
+- INSERT INTO
+
+## CREATE TABLE
+
+The `CreateTable` class of the Sql Statement Builder implements SQL CREATE TABLE
+functionality.
+
+### Usage
+
+1. Create an instance of the `CreateTable` class through the `StatementFactory`
+
+  ```java
+  CreateTable createTable = StatementFactory.getInstance().createTable("tableName");
+  ```
+
+2. Create columns with desired data types using fluent programming
+
+  The column order in a final rendered string will be the same as a sequence of
```

Comment:
"Construct the columns in the order you want them to appear in the created table."

Reply:
fixed